packages/adapter-vercel/index.d.ts

```diff
@@ -4,6 +4,36 @@ type Options = {
 	edge?: boolean;
 	external?: string[];
 	split?: boolean;
-};
+} & Config;
 
 export default function plugin(options?: Options): Adapter;
+
+export interface Config {
```

Comment:
I didn't make this `App.Config` or sth like that because to me it's different to the other `App.X` in that the config is "on its own" and not integrated with any other function call of Kit. Also, if we ever decide you could have multiple adapters per app, multiple `App.Config` definitions would clash.

Reply:
Yeah, that makes sense. Is `Config` right or should it be `RouteConfig` or `FunctionConfig` or something, since `Config` could be used for the adapter config? Counterpoint: better to use the shorter name for something that's potentially used multiple times

Reply:
One disadvantage I just thought of though — if we manage to implement automatic export typing for `load` and `actions` and `data` etc, then it would be a bummer if we couldn't automatically type `config` because we went with `import('some-adapter').Config` rather than `import('@sveltejs/kit').App.RouteConfig`

Reply:
We could always add `App.RouteConfig` later on and deprecate the `Config` export